src/Stryker.Core/Stryker.Core/Language/Language.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Stryker.Core
```

Comment:
The namespace should probably be Stryker.Core.Language if it remains in this location. However I suggest you move the Language enum to Initialisation and change the namespace respectively.